Quaver.Shared/Screens/Edit/Actions/HitObjects/Resnap/EditorActionResnapHitObjects.cs

```diff
@@ -126,7 +126,7 @@ public void Perform()
                     new EditorActionHitObjectsResnappedEventArgs(Snaps, HitObjectsToResnap));
             }
             else
-                NotificationManager.Show(NotificationLevel.Info, $"No notes to resnap");
+                NotificationManager.Show(NotificationLevel.Info, $"No notes resnapped");
```

Comment:
This notification should be removed when pasting. It's fine to keep it when using the resnap feature though.